src/components/picture/picture-area.tsx

```diff
@@ -574,18 +565,19 @@ export class PictureArea extends BaseComponent<IProps, {}> {
       <div className="subsection simulation">
         <div className="picture-area-container">
           { renderScenery() }
-          { renderTrees() }
           { renderRivers(flowPercentage / 25) }
           { renderLake(currentLakeArea) }
           { renderDamn() }
+
+          {/* <FullSizeThing x={200} y={250} e_x={20} e_y={25} w={600} h={340} f={width / 600.0}/> */}
```

Comment:
and this looks like it needs to come out

Reply:
Yep, I missed it. Thanks.